middleman_protocol/middleman_protocol/concent_golem_messages/message.py

```diff
@@ -21,9 +23,16 @@ class NonceAbstractMessage(Message):
         'nonce',
     ] + Message.__slots__
 
+    def slots(self) -> List[List[Union[str, bytes, int]]]:
+        slots = super(NonceAbstractMessage, self).slots()
```

Comment:
You can call `super()` without parameters in Python3.

Reply:
Done